models/migrations/v68.go

```diff
@@ -74,8 +84,16 @@ func reformatAndRemoveIncorrectTopics(x *xorm.Engine) (err error) {
 			delTopicIDs = append(delTopicIDs, topic.ID)
 		}
 	}
+	if err := sess.Commit(); err != nil {
+		return err
+	}
+
+	sess.Init()
```

Comment:
This is necessary if you update xorm to latest version. see https://github.com/go-xorm/xorm/pull/1015

Reply:
But with new version, pgsql integration tests are failed